graphics/ipe-tools/Portfile

```diff
@@ -20,18 +19,20 @@ long_description        The following tools for the Ipe extensible drawing \
 
 homepage                http://ipe.otfried.org/
```

Comment:
Can we make this URL HTTPS? Ideally all URLs should be.

Reply:
done